corehq/apps/accounting/tasks.py

```diff
@@ -896,3 +898,25 @@ def email_enterprise_report(domain, slug, couch_user):
            "You can download the data at the following link: {}<br><br>" \
            "Please remember that this link will only be active for 24 hours.".format(account.name, link)
     send_html_email_async(subject, couch_user.username, body)
+
+
+@periodic_task(run_every=crontab(hour=1, minute=0, day_of_month='1'), acks_late=True)
+def calculate_users_in_all_domains():
+    delete_outdated_domain_user_history(datetime.date.today())
```

Comment:
How did you decide to delete old history?  It might be worth keeping the history around for record keeping, unless you have a reason to be concerned about the size of the table.

Reply:
I guess we aren't running a risk that the table will get that large. I'll take this method out and if we ever have a reason to delete the history, we can deal with it then.